host/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "trouble-host"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
No need to bump

Reply:
Reverted in 23b13fb11d984f4f5b5ae40adab18690d424de47